src/capture_RPiHQ.cpp

```diff
@@ -381,7 +384,7 @@ int calculateTimeToNightTime(const char *latitude, const char *longitude, const
     if (diff < 0)
     {
 		// This assumes tomorrow's nighttime starts same as today's, which is close enough.
-		return(diff + (60*60*24));	// Add one day
+		return(diff + S_IN_DAY);	// Add one day
```

Comment:
to check/understand this line I have to look for declaration and assignment before to be sure that this is no sum of different units. (my brain is to small to remember all the lines before) 
please use diff_s

Reply:
Ooops.  I had fixed this in capture.cpp but not capture_RPiHQ.cpp.  Just fixed it there, too.

Reply:
@AndreasLMeg When you review these changes it looks like you're approving the PR so it can be merged.  Is that your intention, versus just requesting a change?  I'm asking because I thought you wanted to do more testing before we merge the PR.